landlab/components/depth_dependent_cubic_soil_creep/hillslope_depth_dependent_cubic_flux.py

```diff
@@ -181,32 +261,84 @@ def __init__(self,grid,
         self._active_nodes = self.grid.status_at_node != CLOSED_BOUNDARY
 
 
-    def soilflux(self, dt):
+    def soilflux(self, dt, dynamic_dt=False, if_unstable='pass', courant_factor=0.2):
         """Calculate soil flux for a time period 'dt'.
```

Comment:
@mcflugen OK Great! I wasn't sure if I should have been able to use
 
```python
pass
```
 (instead of the string 'pass') so I wanted to make sure that the strings were what you were thinking of!

thanks!